lib/run.js

```diff
@@ -32,10 +32,14 @@ function validateOpts(options) {
     
     validateNumberProp(options, 'rate');
     validateNumberProp(options, 'concurrent');
-    
-    // hack to enable concurrency mode,
-    // since forkmaster expects this to be defined
-    options.rate = options.rate || 0;
+
+    if (options.concurrent) {
+        delete options.rate;
+    }
+
+    if (options.rate < 0 || Math.floor(options.rate * (options.duration / 1000)) < 1) {
+        throw new Error('duration is too low');
```

Comment:
Hmm... this technically means that the rate to duration ratio is not doable.

I was kind of considering running a single test in this case. Presumably, the only intention a user could have for calling run (or using the CLI) is to run a test, which means that expecting at least 1 run is pretty reasonable. Right?